DfE.FindInformationAcademiesTrusts/Pages/Trusts/Details.cshtml

```diff
@@ -16,17 +16,17 @@
             Address
           </dt>
           <dd class="govuk-summary-list__value" data-testid="address">
-            @Model.Trust.Address
+            @Model.TrustDetailsServiceModel.Address
```

Comment:
Naming is hard, and here this feels wrong, but one for us to address in the fullness of time 🤔

Reply:
I could call the property `Trust`? There's no hard rule that says it **has** to be named the same as the type and it would look nicer

Reply:
Renamed